### PR TITLE
Update Deno Syntax to New 1.0 Syntax, 

### PR DESCRIPTION
**Pull request by grant**
This PR fixes some issues with upgrading the `helloworld-deno` sample to `deno@1.0.0-rc2`.
Related #2444 #2456

@averikitsch https://github.com/knative/docs/pull/2456#issuecomment-627676819

Tested locally and on Cloud Run:
https://hellodeno-q7vieseilq-uc.a.run.app

Sorry for the multiple PRs!

### CODE REVIEW

File: community/samples/serving/helloworld-deno/README.md [hunk 1]
```diff
@@ -29,47 +29,37 @@ cd knative-docs/docs/serving/samples/hello-world/helloworld-deno
...
    ```
 
 1. Create a new file named `main.ts` and paste the following script:
 
    ```ts
    import { serve } from "./deps.ts";
-
-   const PORT = 8080;
+   import "https://deno.land/x/dotenv/mod.ts";
+
```
Comment (mattmoor) [inline community/samples/serving/helloworld-deno/README.md:40]: Remove trailing whitespace:
```suggestion

```

File: community/samples/serving/helloworld-deno/README.md [hunk 2]
```diff
@@ -29,47 +29,37 @@ cd knative-docs/docs/serving/samples/hello-world/helloworld-deno
...
    import { serve } from "./deps.ts";
-
-   const PORT = 8080;
+   import "https://deno.land/x/dotenv/mod.ts";
+   
+   const PORT = Deno.env.get('PORT') || 8080;
    const s = serve(`0.0.0.0:${PORT}`);
    const body = new TextEncoder().encode("Hello Deno\n");
-
+
```
Comment (mattmoor) [inline community/samples/serving/helloworld-deno/README.md:44]: Remove trailing whitespace:
```suggestion

```

File: community/samples/serving/helloworld-deno/README.md [hunk 3]
```diff
@@ -29,47 +29,37 @@ cd knative-docs/docs/serving/samples/hello-world/helloworld-deno
...
-
-    # Added to ENTRYPOINT of base image.
-    CMD ["--allow-net", "main.ts"]
-    ```
+1. Create a new file named `Dockerfile` and copy the code block below into it.
+
+   ```docker
+   FROM hayd/alpine-deno:1.0.0-rc2
+   WORKDIR /app
+
```
Comment (mattmoor) [inline community/samples/serving/helloworld-deno/README.md:56]: Remove trailing whitespace:
```suggestion

```

File: community/samples/serving/helloworld-deno/README.md [hunk 4]
```diff
@@ -29,47 +29,37 @@ cd knative-docs/docs/serving/samples/hello-world/helloworld-deno
...
-    ```
+1. Create a new file named `Dockerfile` and copy the code block below into it.
+
+   ```docker
+   FROM hayd/alpine-deno:1.0.0-rc2
+   WORKDIR /app
+ 
+   # These steps will be re-run upon each file change in your working directory:
+   COPY . ./
+
```
Comment (mattmoor) [inline community/samples/serving/helloworld-deno/README.md:59]: Remove trailing whitespace:
```suggestion

```

**Comment by grant** [comment]
I'm not sure why the tests are pending.

**Comment by averikitsch** [comment]
Sorry about the past PRs. I forgot these auto-merge.

**Comment by grant** [comment]
Unrelated failures.
test/e2e TestSampleApp/kotlin 2m30s

@averikitsch Do you know how to re-test or get this through?

**Pull request merged**